mfr/extensions/image/export.py

```diff
@@ -23,7 +25,14 @@ def export(self):
             'max_size_h': max_size[1],
         })
         try:
-            image = Image.open(self.source_file_path)
+            if self.ext in ['.psd']:
+                # silence warnings from psd-tools
```

Comment:
Warnings are custom thrown by the library. If there is an error, that still is caught. warnings look something like this: 
/usr/local/lib/python3.5/site-packages/psd_tools/decoder/image_resources.py:138: UserWarning: ICC profile is not handled; colors could be incorrect. Please build PIL or Pillow with littlecms/littlecms2 support.

I remember doing some research into this when i originally did this PR, but Its been quite awhile and i can't remember much besides coming to the conclusion that it would be okay.
Warnings are silence so they dont spam the logs.

Reply:
I see. Warnings are fine since they are not exceptions. Your comment in the code also helps. We do want to catch exceptions thrown by `psd-tools` but maybe in another ticket so this is not a blocker.